config/data/diff_config_1.json

```diff
@@ -2,7 +2,7 @@
     "modules": [
         {
             "name": "my-module",
-            "executable_path": "path/to/my-module",
+            "executable_path": ".",
```

Comment:
This and the other `executable_path` in `diff_config_2.json` had to be real paths; otherwise the validation error would cause issues when inspecting diffs between configs in `diff_test.go`.